moveit_plugins/moveit_ros_control_interface/src/controller_manager_plugin.cpp

```diff
@@ -596,11 +596,11 @@ class Ros2ControlManager : public moveit_controller_manager::MoveItControllerMan
     dependency_map_reverse_.clear();
     for (auto& controller : result->controller)
     {
-      if (controller.chain_connections.size() > 1)
+      if (isActive(controller) && controller.chain_connections.size() > 1)
```

Comment:
I don't have a way to verify this so any [test](https://github.com/thomaspeyrucain/moveit2/blob/a62d02d1172dba4bf90cf7da58e0d6af73866a4b/moveit_plugins/moveit_ros_control_interface/test/test_controller_manager_plugin.cpp) you could add would be extremely helpful. 

Example scenario:
```
arm_1_JTC active
arm_2_JTC active
arm_3_chained inactive or active
```

Can you confirm that when `controller = arm_1_JTC`, `controller.chain_connections.size()` == 0
and when `controller = arm_3_chained`, `controller.chain_connections.size()` == 1?